packages/craftcms-webpack/index.js

```diff
@@ -0,0 +1,11 @@
+const glob = require('glob');
+const path = require('path');
+const fetchConfigs = (
```

Comment:
Was it not possible to put this in its own file having the "base path" passed into the constructor? With there default being the `module.parent.path`?